src/database/t/test_song_segment.py

```diff
@@ -0,0 +1,34 @@
+from database.song_segment import SongSegment
+from database.storinator import Storinator
+import datetime
+
+
+def test_implements_Storinator():
+    ss = SongSegment()
+    assert isinstance(ss, Storinator)
+
+
+def test_database_name():
+    ss = SongSegment()
+    assert ss._dbname == 'song_segmentation'
```

Comment:
I'd change this to test collection name instead of the db name. As that actually is what you're testing ;)